SNIPS/snip-6.md

```diff
@@ -2,7 +2,7 @@
 snip: 6
 title: Standard Account Interface
 authors: Martín Triay <martriay@gmail.com>, Julien Niset <julien@argent.xyz>, Eric Nordelo <eric.nordelo39@gmail.com>, Sergio Garcia <sergio@argent.xyz>, Yoav Gaziel <yoav.gaziel@braavos.app>
-discussions-to: https://community.starknet.io/t/snip-starknet-standard-account/95665
+discussion-to: https://community.starknet.io/t/snip-starknet-standard-account/95665
```

Comment:
this is not a typo. Could you undo this change so that we get the first typo fix merged? Thanks!